### PR TITLE
Fixed wrong tcp data field setting which made cksum test fail. Small changes in test itself

### PR DESCRIPTION
**Pull request by dkolistratova**
#323 fix

### CODE REVIEW

File: common/common.go [hunk 1]
```diff
@@ -74,6 +74,7 @@ const (
 	IPv6Len    = 40
 	ICMPLen    = 8
 	TCPMinLen  = 20
+	TCPDataOff = 0x50
```
Comment (mbulatova) [inline common/common.go]: It is actually TCPMinDataOffset

File: packet/packet.go [hunk 2]
```diff
@@ -554,8 +553,8 @@ func InitEmptyIPv4TCPPacket(packet *Packet, plSize uint) bool {
 	packet.GetIPv4NoCheck().TotalLength = SwapBytesUint16(uint16(IPv4MinLen + TCPMinLen + plSize))
 
 	packet.ParseL4ForIPv4()
-	packet.GetTCPNoCheck().DataOff = packet.GetTCPNoCheck().DataOff | 0x50 // TODO check
-
+	packet.GetTCPNoCheck().DataOff = TCPDataOff
+	packet.Data = unsafe.Pointer(uintptr(packet.L4) + uintptr(((*TCPHdr)(packet.L4)).DataOff&0xf0)>>2)
```
Comment (gshimansky) [inline packet/packet.go]: Why not use packet.GetTCPNoCheck().DataOff here?

File: packet/packet.go [hunk 3]
```diff
@@ -626,16 +625,15 @@ func InitEmptyIPv6TCPPacket(packet *Packet, plSize uint) bool {
...
 	packet.ParseL3()
 	packet.GetIPv6NoCheck().Proto = TCPNumber
 	packet.GetIPv6NoCheck().PayloadLen = SwapBytesUint16(uint16(TCPMinLen + plSize))
 	packet.GetIPv6NoCheck().VtcFlow = SwapBytesUint32(0x60 << 24) // IP version
 
 	packet.ParseL4ForIPv6()
-	packet.GetTCPNoCheck().DataOff = packet.GetTCPNoCheck().DataOff | 0x50
-
+	packet.GetTCPNoCheck().DataOff = TCPDataOff
+	packet.Data = unsafe.Pointer(uintptr(packet.L4) + uintptr(((*TCPHdr)(packet.L4)).DataOff&0xf0)>>2)
```
Comment (gshimansky) [inline packet/packet.go]: Why not use packet.GetTCPNoCheck().DataOff here?

File: common/common.go (hunk 1 above)
Comment (gshimansky) [reply common/common.go]: I agree.

Comment (ifilippov) [reply common/common.go]: If you start to add new constants not connected with length, can you
1) start const section
2) add constants
0x45 // Ipv4, IHL = 5 (min header len) - 610 line
SwapBytesUint32(0x60 << 24) // IP version - 632 line
and others.

File: test/stability/testCksum/testCksum.go [hunk 4]
```diff
@@ -47,7 +47,7 @@ const (
 )
 
 var (
-	totalPackets uint64 = 10
+	totalPackets uint64 = 32
```
Comment (ifilippov) [inline test/stability/testCksum/testCksum.go]: Was this tested both at physical machines and virtual machines?

Comment (dkolistratova) [reply test/stability/testCksum/testCksum.go]: yes. 32 is more logical cause its the burst size.

**Comment by dkolistratova** [comment]
failed due to dpdk site unavailable. Checked locally, works.

**Pull request merged**